src/blueapi/messaging/stomptemplate.py

```diff
@@ -101,7 +101,9 @@ def __init__(
         self._subscriptions = {}
 
     @classmethod
-    def autoconfigured(cls, config: StompConfig) -> MessagingTemplate:
+    def autoconfigured(cls, config: StompConfig | None) -> MessagingTemplate | None:
+        if config is None:
+            return None
```

Comment:
Should: Similar to `AmqClient`, I think accepting `None` in the signature but immediately rejecting it in the body is confusing. I'd prefer that this only accept `config: StompConfig` and the code calling `autoconfigured` performs the `None`-check